### DISCUSSION TITLE
osu! minimises on focus loss

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#20832] [Q&A]
Whenever i tab out of the game or move my cursor to another screen, the game minimises. This is especially annoying when trying to drop beatmaps or skins into the window. The issue only happens in fullscreen and borderless mode, so i always gotta switch to windowed mode for that. This is different to the way stable works, when i move my cursor to another screen or alt tab to the file manager, the game stays maximised. The screen mode also doesn't matter there.

I'm running linux with LXDE rn, but this also happened on KDE and GNOME before. Afaik this doesn't happen on windows, at least in borderless mode, not sure about fullscreen. Maybe someone else can test that.

**Comment by SupDos** [comment]
Has been talked about in this discussion https://github.com/ppy/osu/discussions/17172, and stable recently got an update that added a toggle to change auto-minimize behavior when in fullscreen, so hopefully it means lazer will also be able to stay visible when losing focus at some point
